src/module.c

```diff
@@ -6704,6 +6721,50 @@ int RM_SetModuleUserACL(RedisModuleUser *user, const char* acl) {
     return ACLSetUser(user->user, acl, -1);
 }
 
+/* Check if the command can be executed by the user created through the redis module
+ * interface, according to the ACLs associated to it.
+ *
+ * If the user can execute the command, REDISMODULE_OK is returned, otherwise
+ * REDISMODULE_ERR is returned. */
+int RM_ACLCheckCommandPerm(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
```

Comment:
Redis Module API tends to use longer names and avoid shorthand, consider `RM_ACLCheckCommandPermissions`.

Also, I am not sure about the coupling with `RedisModuleCtx` here. Doesn't it make more sense to take a `RedisModuleUser` here, and have a separate function that can extract `RedisModuleUser` from the context?